### DISCUSSION TITLE
Sentry integratiton/support

### DISCUSSION THREAD

**Discussion by mparpaillon** [analogjs/analog#1480] [Q&A]
### Which scope/s are relevant/related to the feature request?

create-analog

### Information

I'm looking into Sentry and sadly Analog isn't there. 

![Capture d’écran 2024-12-05 à 09 40 50](https://github.com/user-attachments/assets/cb4d3b5e-4fcd-468f-af11-884ef9d02c79)

There is Angular, NodeJS and even a [tRPC middleware](https://docs.sentry.io/platforms/javascript/guides/express/configuration/integrations/trpc/) but the first step of the Angular integration tutorial is to add some code in `main.ts`. But there's also `main.server.ts`...

So I'm wondering. What would be your recommendations as to how properly integrate this kind of error tracking tool to an Analog project?

Thanks a lot. Analog is awesome <3

### I would be willing to submit a PR to fix this issue

- [ ] Yes
- [X] No

**Comment by louiiuol** [comment]
Hi there 🙌

I've tried to integrate Sentry on my analog application, but it's partially works..
First, I tried to create separate projects on sentry:
- An angular preset for the frontend, which works pretty well
- A node preset that fails to setup; I followed the documentation and placed Sentry initialisation inside `trpc.ts` (where all middleware are declared). But the errors won't show in my project.
For example, when an error throws from the backend, I can see the error only in frontend project, but none in my backend (and tracing doesn't work on trpc)

I've also tried to do a node vanilla config, unsuccesfuly .. not sure if it's actually possible, nor how it should be done...

Any guidance on this would be greatly appreciated. 

Thanks 🙏
